### DISCUSSION TITLE
Random missing conversions when using Redis queue

### DISCUSSION THREAD

**Discussion by Vercoutere** [spatie/laravel-medialibrary#3234] [Q&A]
We run a script that imports a bunch of data, and updates media (images) related to that data.
Everything works great when we run our script with conversion queueing disabled. When we enable pushing conversions to the queue, however, we always end up with models that do not have any conversions.

If we run our script multiple times with the same data, the models that are missing conversions are different.
It appears as if random conversion jobs are being dropped from the queue.

We use Horizon to manage our queue and don't get any failed jobs in there.

If I regenerate all conversions using the artisan CLI, that works flawlessly, and no conversions are missing anywhere.

I'm not sure how to further debug this issue as it appears to be random from my perspective. Any help would be appreciated.

**Comment by chimit** [comment]
@Vercoutere did you manage to find the reason?

I have the same problem. I can add that tasks look fully successful and completed in Horizon. Only conversions are missing in my DigitalOcean Spaces bucket (the original files are there). Doesn't look like a Horizon problem.
